.buildkite/verify-yaml.sh

```diff
@@ -5,7 +5,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")/.."
 
 gcloud container clusters get-credentials cloud --zone us-central1-f --project sourcegraph-dev
 
-kubectl apply --dry-run --validate --recursive -f base/ --context=gke_sourcegraph-dev_us-central1-f_cloud
+kustomize build ./base/. | kubectl apply --dry-run --validate --context=gke_sourcegraph-dev_us-central1-f_cloud -f -
```

Comment:
Why the change in the CI for the base cluster? I assume this is technically equivalent to before, but I could see the motivation be that we want to test a command that is more directly what we ask users to run.

Would it also make sense to add CI commands for `new/base`?

Reply:
> Why the change in the CI for the base cluster? 

this is due to the[ kustomization files added to the original base](https://github.com/sourcegraph/deploy-sourcegraph/tree/bee/newBase/base) that requires kustomize to run, but since we will be setting up a separate repo for the new base, this change will not be necessary.

> Would it also make sense to add CI commands for new/base?

It makes sense to me! I've created a new issue for creating tests for the new base https://github.com/orgs/sourcegraph/projects/290?pane=issue&itemId=18056795